pgml-extension/src/bindings/transformers.py

```diff
@@ -110,22 +110,38 @@ def transform(task, args, inputs):
     return json.dumps(pipe(inputs, **args), cls=NumpyJSONEncoder)
 
 
-def embed(transformer, text, kwargs):
+def embed(transformer, inputs, kwargs):
+    
+    try:
+        inputs = json.loads(inputs)
+    except json.decoder.JSONDecodeError:
```

Comment:
What is the known case that this is handling?

Reply:
Without passing any extra arguments, I couldn't think of a way of knowing wether inputs was a string or a JSON string. Thought the simplest way was trying to convert it into a python object.